### DISCUSSION TITLE
Procedure for unattended action items?

### DISCUSSION THREAD

**Discussion by JohnStarich** [graphql/composite-schemas-wg#76] [Q&A] [answered]
I'm not super familiar with WG processes. Is there something we should do for action items that are untouched for a couple months?

**Comment by benjie** [comment, marked as answer]
Nothing in particular; since progressing action items is done on a voluntary basis it can sometimes take a while for people to find time to progress them. If the assignee has decided they don't want to progress it any further then it can be closed, similarly if the action is complete then it can be closed. Closing items takes place at the WG meeting. The main GraphQL Spec WG has items open for over a year quite frequently.
